SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -1223,6 +1223,13 @@ FReply FSpatialGDKEditorToolbarModule::OnStartCloudDeployment()
 	{
 		if (CloudDeploymentConfiguration.bGenerateSchema)
 		{
+			if (SpatialGDKEditorInstance->FullScanRequired())
+			{
+				FMessageDialog::Open(EAppMsgType::Ok, FText::FromString(TEXT("A full schema generation is required before being able to start a cloud deployment. Please press the Schema button before starting a cloud deployment.")));
```

Comment:
Maybe I am overthinking this, but if I was a first-time user, I would think a full schema gen is required before *every* cloud deploy from the wording here.
"Full schema generation must be run at least once before starting a cloud deployment. ..."?
Someone else will probably be able to convey this better.

Reply:
true, it's a bit vague right now. i'll also ask our tech writers to see what they think of how to phrase it